lighthouse-core/audits/screenshot-thumbnails.js

```diff
@@ -78,9 +78,12 @@ class ScreenshotThumbnails extends Audit {
     ]).then(([speedline, ttfi, ttci]) => {
       const thumbnails = [];
       const analyzedFrames = speedline.frames.filter(frame => !frame.isProgressInterpolated());
+      const maxFrameTime =
+        speedline.complete ||
```

Comment:
seems like we could just ALSO put speedline.complete in the L86 math.max(), yeah? 
 
semantically maxFrameTime is the second half of this, but not realllly .complete.